dietpi/misc/dietpi-nordvpn

```diff
@@ -336,6 +336,7 @@ _EOF_
 
 			elif [[ $G_WHIP_RETURNED_VALUE == 'Update' ]]; then
 
+				G_EXEC_DESC='Removing old NordVPN server configs' G_EXEC rm -Rf /etc/openvpn/ovpn_{tcp,udp}/*.nordvpn.com.{tcp,udp}.ovpn
```

Comment:
Let's do this only after successful download of the new archive. E.g. if something has changed, the download URL or the specific CDN target server is down, user will not end up with broken NordVPN, but can continue with the previous list.

Reply:
Ah that make sense. But than we would need to analyze the result of `G_EXEC curl -sSfL https://downloads.nordcdn.com/configs/archives/servers/ovpn.zip -o 171.zip`. Isn't it? How would you do that? Checking if the zip file exist? Or is there a return code given that can be used?